check.py

```diff
@@ -195,21 +195,14 @@ def fail_if_not_contained(actual, expected):
           raise Exception('wasm interpreter error: ' + err) # failed to pretty-print
         raise Exception('wasm interpreter error')
 
-print '\n[ checking binaryen-shell... ]\n'
```

Comment:
All the programs are using the same command-line support, so they either all work on none of them work ;-)

I can add a separate command-line only test, but that seemed like overkill.